File: .sdk-repos/orchestration-cluster-api-go/examples/adhoc.go
// Ad-hoc sub-process operations: activate activities inside a running ad-hoc
// sub-process instance.
package examples

import (
	"context"

	camunda "github.com/camunda/orchestration-cluster-api-go"
	openapi "github.com/camunda/orchestration-cluster-api-go/client"
)

func activateAdHocSubProcessActivitiesExample(ctx context.Context, client *camunda.CamundaClient) error {
	// region ActivateAdHocSubProcessActivities
	instruction := openapi.NewAdHocSubProcessActivateActivitiesInstruction(
		[]openapi.AdHocSubProcessActivateActivityReference{
			*openapi.NewAdHocSubProcessActivateActivityReference("review-task"),
		})

	return client.ActivateAdHocSubProcessActivities(ctx,
		openapi.MustElementInstanceKey("2251799813685360"), *instruction)
	// endregion ActivateAdHocSubProcessActivities
}

File: .sdk-repos/orchestration-cluster-api-php/examples/adhoc.php
<?php

/**
 * Compilable usage examples for ad-hoc sub-process operations.
 */

declare(strict_types=1);

namespace Camunda\Orchestration\Examples;

use Camunda\Orchestration\CamundaClient;

// region ActivateAdHocSubProcessActivities
/**
 * Activate activities within an ad-hoc sub-process.
 */
function activate_ad_hoc_sub_process_activities(CamundaClient $client, string $adHocSubProcessInstanceKey, \Camunda\Orchestration\Api\Model\AdHocSubProcessActivateActivitiesInstruction $adHocSubProcessActivateActivitiesInstruction): void
{
    $client->activateAdHocSubProcessActivities($adHocSubProcessInstanceKey, $adHocSubProcessActivateActivitiesInstruction);
}
// endregion ActivateAdHocSubProcessActivities
